doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,267 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing Circuits
+==================
+
+PennyLane offers the :mod:`~.io` module to import quantum circuits and operations that were
+constructed outside of PennyLane. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
+`OpenQASM <https://docs.quantum.ibm.com/build/interoperate-qiskit-qasm2>`_, and `Quil
+<https://docs.rigetti.com/qcs/guides/quil>`_.
+
+.. note::
+
+    To import a quantum circuit defined using a particular framework, you will need to install the
+    corresponding PennyLane plugin for that framework. More information about PennyLane plugins is
+    available on the `plugins <https://pennylane.ai/plugins.html>`_ page.
+
+Importing Quantum Circuits
+--------------------------
+
+Importing quantum circuits can help you take advantage of PennyLane's various optimization,
+visualization, and interoperability features for existing circuits. For example, you can take a
+``QuantumCircuit`` from Qiskit, import it into PennyLane, and then apply a `circuit-cutting transform
+<https://pennylane.ai/qml/demos/tutorial_quantum_circuit_cutting/>`_ to reduce the number of qubits
+required to implement the circuit. You could also :ref:`compile <intro_ref_compile_circuits>` the
+circuit using `Catalyst <https://docs.pennylane.ai/projects/catalyst/en/stable/index.html>`__ or
+execute the circuit remotely using a supported hardware provider.
+
+Qiskit
+~~~~~~
+
+To import a quantum circuit from Qiskit, you must first install the `PennyLane-Qiskit
+<https://docs.pennylane.ai/projects/qiskit/en/stable/>`__ plugin. This can be done by running:
+
+.. code-block::
+
+    pip install pennylane-qiskit
+
+Now, suppose we define a Qiskit `QuantumCircuit
+<https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit>`__ as follows:
+
+.. code-block:: python
+
+    from qiskit import QuantumCircuit
+
+    qk_circuit = QuantumCircuit(2, 1)
+    qk_circuit.h(0)
+    qk_circuit.cx(0, 1)
+    qk_circuit.measure_all()
+
+We can convert the ``QuantumCircuit`` into a PennyLane quantum function using:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_qk = qml.from_qiskit(qk_circuit)
+
+Above, the :func:`from_qiskit` function converts a ``QuantumCircuit`` into a PennyLane template.
+This template can then be called from inside a QNode to generate a PennyLane circuit:
+
+.. code-block:: python
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_qk():
+        pl_template_from_qk(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.expval(qml.Z(1))
```

Comment:
```suggestion
        measurements = pl_template_from_qk(wires=[0, 1])
        return [qml.expval(m) for m in  measurements]
```

I think we should either do this, or remove the `qml.measure_all` from the circuit. It doesn't actually make a difference for this circuit, but the way this is done here, we do a measurement in the Y basis after a projective measurement into the Z basis. I think this is bad practice and shouldn't be in our examples. What are other people's thoughts on this?

Reply:
Yes +1 to remove the `measure_all`

Reply:
@trbromley To clarify, are you suggesting to remove `measure_all` or adopt @lillian542's suggestion? Or both?

Reply:
We can just remove the `measure_all`, and then I think the current makes sense, i.e.
```
        pl_template_from_qk(wires=[0, 1])
        return qml.expval(qml.Y(0)), qml.expval(qml.Z(1))
```
(but correct me if I'm wrong)

Reply:
Okay, done!